package.json

```diff
@@ -40,7 +40,7 @@
     "hex-rgb": "^5.0.0",
     "highcharts": "^11.4.8",
     "lodash.throttle": "^4.1.1",
-    "nuxt": "^3.13.2",
+    "nuxt": "^3.14.1592",
```

Comment:
They're doing a pi joke with their version numbers...

Reply:
not sure how i feel about this one :laughing: